src/extension.ts

```diff
@@ -229,21 +227,28 @@ export function activate(context: vscode.ExtensionContext) {
 export function deactivate() {
 }
 
-function activateDps(context: vscode.ExtensionContext) {
-    let telemetryReporter = new TelemetryClientWrapper("DPS.");
-    let outputChannel = createAzExtOutputChannel("Device Provisioning Service", "azure-iot-toolkit");
+function initializeTreeView(context: vscode.ExtensionContext) {
+    const telemetryReporter = new TelemetryClientWrapper("IoTHub.");
+    const outputChannel = createAzExtOutputChannel("Azure IoT Hub Toolkit", "azure-iot-toolkit");
```

Comment:
same as above the dps and iot hub tree view use the same output channel?

Reply:
Yes~